libnmstate/__init__.py

```diff
@@ -1,18 +1,21 @@
 #
-# Copyright 2018 Red Hat, Inc.
+# Copyright (c) 2018-2019 Red Hat, Inc.
+#
+# This file is part of nmstate
```

Comment:
Why is this needed?
IMO we should also not limit ourself to host our sources on *github*.